queue_services/entity-filer/src/entity_filer/filing_processors/correction.py

```diff
@@ -0,0 +1,52 @@
+# Copyright © 2019 Province of British Columbia
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""File processing rules and actions for the correction filing."""
+from typing import Dict
+
+import pytz
+from legal_api.models import Comment, Filing
+
+
+def process(correction_filing: Filing, filing: Dict):
+    """Render the correction filing onto the business model objects."""
+    local_timezone = pytz.timezone('US/Pacific')
+
+    # link to original filing
+    original_filing = Filing.find_by_id(filing['correction']['correctedFilingId'])
+    original_filing.parent_filing = correction_filing
+
+    # add comment to the original filing
+    original_filing.comments.append(
+        Comment(
+            comment=f'This filing was corrected on '
+                    f'{correction_filing.filing_date.astimezone(local_timezone).date().isoformat()}.',
+            staff_id=correction_filing.submitter_id
+        )
+    )
+
+    # add comment to the correction filing
+    correction_filing.comments.append(
+        Comment(
+            comment=filing['correction']['comment'],
+            staff_id=correction_filing.submitter_id
+        )
+    )
+
+    # set correction filing to PENDING, for manual intervention
+    # - include flag so that listener in Filing model does not change state automatically to COMPLETE
+    correction_filing._status = Filing.Status.PENDING.value  # pylint: disable=protected-access
```

Comment:
Consider a new state for corrections, as PENDING for other filings means awaiting payment.